lib/node_modules/@stdlib/math/base/complex/divide/docs/repl.txt

```diff
@@ -0,0 +1,40 @@
+
+{{alias}}( [out,] re1, im1, re2, im2 )
+    Divides two complex numbers.
+
+    Parameters
+    ----------
+    out: Array|TypedArray|Object
```

Comment:
This is missing an `(optional)` annotation. Sorry for not pointing this out earlier. For the REPL text guide, see https://github.com/stdlib-js/stdlib/blob/develop/docs/repl_text.md.

```
out: Array|TypedArray|Object (optional)
```